system-apps/password-reset-app/webClient/src/environments/environment.prod.ts

```diff
@@ -0,0 +1,27 @@
+
```

Comment:
You can delete the environments folders. Its actually not used by angular components, only angular websites. apps are just components.